cmd/kwild/config/default_config.toml

```diff
@@ -34,20 +34,14 @@
 grpc_listen_addr = "localhost:50051"
 
 # TCP address for the KWILD App's HTTP server to listen on
-http_listen_addr = "localhost:8080"
+http_listen_addr = "0.0.0.0:8080"
```

Comment:
Oh, this file isn't actually "used" anywhere, but the default from `Default()` config is still `localhost:8080`.  I should change this line back.